unit_tests/multiphase/test_momflux.cpp

```diff
@@ -291,10 +295,44 @@ class MassMomFluxOpTest : public MeshTest
             const auto& rf = advrho_f(lev).const_array(mfi);
             const auto& vel = velocity(lev).const_array(mfi);
             const auto& dqdt = conv_term(lev).const_array(mfi);
-            check_accuracy(
-                dir, tol, m_rho1, m_rho2, m_vel, dx, problo, varr, bx, um, vm,
+            // Populate error scratch field
+            auto error = error_fld(lev).array(mfi);
+            get_accuracy(
+                error, dir, m_rho1, m_rho2, m_vel, dx, problo, varr, bx, um, vm,
                 wm, rf, vel, dqdt);
         });
+
+        // Label 0's with variable names so gtest output is decipherable
+        constexpr amrex::Real umac_check = 0.0;
+        constexpr amrex::Real vmac_check = 0.0;
+        constexpr amrex::Real wmac_check = 0.0;
+        constexpr amrex::Real u_check = 0.0;
+        constexpr amrex::Real v_check = 0.0;
+        constexpr amrex::Real w_check = 0.0;
+        constexpr amrex::Real advrho_check = 0.0;
+        constexpr amrex::Real dqdt_check = 0.0;
+
+        // Check error in each mfab
+        for (int lev = 0; lev < repo.num_active_levels(); ++lev) {
+            // Sum error for each component individually
+            amrex::Real err_lev = error_fld(lev).max(0);
+            // Check error
+            EXPECT_NEAR(err_lev, umac_check, tol);
```

Comment:
want to just make all these 1 liners? `EXPECT_NEAR(error_fld(lev).max(0), umac_check, tol);`

Reply:
great idea. I'll do that.